i18n/fr.json

```diff
@@ -218,5 +218,9 @@
 
   "questions": {
     "other": "Questions?"
+  },
+
+  "search": {
+    "other": "Search"
```

Comment:
@davidejones - is this a file that gets sent to transifex?

Reply:
@mstbbs no it won't get sent to transifex we only send source files, so the `en.json` one will get sent. 

The reason that I've modified both files is that i was under the impression if i don't add it to the fr.json then hugo errors because its trying to lookup strings that don't exist when building the french site. 

I just tested this theory though and it gave me no issues so I must have been thinking of something else :confused: